test/integration/controller/jobs/jobset/jobset_controller_test.go

```diff
@@ -490,6 +491,83 @@ var _ = ginkgo.Describe("JobSet controller", ginkgo.Ordered, ginkgo.ContinueOnFa
 				gomega.Expect(replica2.Labels).ShouldNot(gomega.HaveKey("label1"))
 			})
 		})
+
+		ginkgo.It("Should allow to create jobset with one replicated job replica count 0", func() {
```

Comment:
Mybe the test should not be in this `When`, it's not related to admission checks

Reply:
I would also be leaning towards a dedicated `when`. we could call it "basic setup".

Reply:
done